### ISSUE TITLE
FG/BG color contrast is insufficient on Buttons with type="Secondary " (light mode)

### ISSUE THREAD

**Issue by shawninder** [tjikko-studio/components#229]
Lighthouse is complaining:

![Screen Shot 2021-11-11 at 13 50 24](https://user-images.githubusercontent.com/484730/141345566-3216d639-24f2-4156-a175-0a698804f7e7.jpg)

[Learn more](https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=devtools)
